core/quantity/src/test/BearingAzimuth.test.ts

```diff
@@ -2,7 +2,7 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import { assert, expect } from "chai";
+import { beforeEach, describe, expect, it } from "vitest";
```

Comment:
Any reason for the `assert` to `expect` change?

Reply:
less imports - in vitest, chai and jest both have test assertions that use `expect`, and we can move towards using one syntax. I see a lot more usage of `expect` than `assert` in core-quantity in particular

Reply:
Just saw @eringram's older comment who noticed the same thing. The PR would be cleaner without the change, but I understand coalescing around one or the other.

Reply:
unless a test is explicitly testing that the types match, using `toEqual` is fine for many cases. When a user writes a test that tests types, they'll probably learn the hard way that `toEqual` will not do the job and can discover `toStrictEqual`.